pkg/utils/ksyms/safe_ksym_table.go

```diff
@@ -0,0 +1,49 @@
+package ksyms
+
+import (
+	"sync"
+
+	"github.com/aquasecurity/libbpfgo/helpers"
+)
+
+type safeKsymbolTable struct {
+	helpers.KernelSymbolTable
+	l *sync.Mutex
```

Comment:
Why not using `RWMutex`?

Reply:
Because it's an interface underneath which we can't know if a read won't include a write in the internal map.

Reply:
I see what you are saying.
LGTM